handsontable/src/dataMap/metaManager/mods/dynamicCellMeta.js

```diff
@@ -49,8 +49,10 @@ export class DynamicCellMetaMod {
    * To boost performance, the extending process is triggered only once per one slow Handsontable render cycle.
    *
    * @param {object} cellMeta The cell meta object.
+   * @param {object} options Execution options for the `extendCellMeta` method.
+   * @param {boolean} [options.executeCellsFunction] If `false`, disables the execution of the `cells` function. Default: `true`.
    */
-  extendCellMeta(cellMeta) {
+  extendCellMeta(cellMeta, options = { executeCellsFunction: true }) {
```

Comment:
With the suggestions I put above, skipping executions could look like this
```js
  extendCellMeta(cellMeta) {
    if (cellMeta._skipMetaExtension) {
       cellMeta._skipMetaExtension = false; 

       return;
    }
```
... then the changes below of this method could be safely reverted - I guess.

Reply:
I did it slightly differently. In `MetaManager.getCellMeta` I put:

```js
    if (!options.skipMetaExtension) {
      this.runLocalHooks('afterGetCellMeta', cellMeta);
    }
```